### DISCUSSION TITLE
Different styles for Desktop and Mobile.

### DISCUSSION THREAD

**Discussion by jay-softsages** [GrapesJS/grapesjs#4905] [Q&A]
I need to give different styles for Desktop and Mobile.

I also referred to the link below -
[https://github.com/GrapesJS/grapesjs/issues/3056
](https://github.com/GrapesJS/grapesjs/issues/3056
)but the responsive design feature is only applicable to the webpage plugin and not to the MJML plugin.
Any idea how to achieve this or is this a feature that needs to be added?
Please help!

**Comment by artf** [comment]
The responsive design with the MJML plugin is handled by MJML itself in that case and the editor only applies values on available mjml properties

**Reply by jay-softsages** [reply]
Thanks for the reply @artf.

I understand that MJML has its own responsive feature but there are some cases where I want to hide some components or change the background color on mobile. 
In such cases how can I provide these features to the users?
The webpage plugin adds a media query I guess for different screen sizes so how can I achieve this same functionality for the MJML plugin?
